app/lib/shared/analyzer_client.dart

```diff
@@ -82,6 +82,8 @@ class AnalysisView {
   DateTime get timestamp => _data.timestamp;
   AnalysisStatus get analysisStatus => _data.analysisStatus;
 
+  DartPlatform get platform => _summary?.platform;
```

Comment:
Where is that `DartPlatform` coming from? I cannot see any class definition. Is this from the pana?

Reply:
It is from `pana`: https://github.com/dart-lang/pana/blob/master/lib/src/platform.dart#L30